src/solc_0.8/assetERC1155/AssetERC1155.sol

```diff
@@ -13,8 +18,10 @@ contract AssetERC1155 is AssetBaseERC1155 {
         address bouncerAdmin,
         IAssetERC721 assetERC721,
         uint8 chainIndex
-    ) external {
+    ) external initializer() {
```

Comment:
`initializer` not `initializer()` https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable